docs/reference/api/usb/USBMouse.md

```diff
@@ -0,0 +1,74 @@
+## USBMouse
+
+<span class="images">![](https://os.mbed.com/docs/v5.9/feature-hal-spec-usb-device-doxy/class_u_s_b_mouse.png)<span>USBMouse class hierarchy</span></span>
+
+You can use the USBMouse interface to emulate a mouse over the USB port. You can choose relative or absolute coordinates and send clicks, button state and scroll wheel movements. If you need keyboard or keyboard and mouse functionality, please see the [USBKeyboard](USBKeyboard.html) and [USBMouseKeyboard](USBMouseKeyboard.html) classes.
```

Comment:
What are some best practices for using USBMouse? What are some concepts you need to understand to use it?